doc/sphinx/extract_rst.py

```diff
@@ -110,6 +111,12 @@ def extract_rst_from_cpp(inname, outname=None):
         r = r.expandtabs(4)
         output += r + '\n'
 
+    # Add string 'Child of: …'
+    output = doc_enh.add_child_of(output, outname)
+    if '-graphs' in sys.argv:
+        # Add dependency graph
```

Comment:
```suggestion
        # Add dependency graph
        print("Adding dependency graphs.")
```

Reply:
Good idea, added a print statement. Not in `extract_rst.py` but in `documentation_enhancements.py` so that only the output mentions real created graphs.

The thing with the `class` not being tied to the class name is an issue of the website css. Unfortunately i have deployed the website related branch just yesterday :-D But yes, this can be solved. Try:
[current.zip](https://github.com/widelands/widelands/files/7845293/current.zip)

Reply:
Related website PR: https://github.com/widelands/widelands-website/pull/370